### PR TITLE
Support Alpine 3.8

### LINKED ISSUE #228: alpine 2.8 support
It would be nice to have alpine 3.8 as image base for newer rubies > 2.5
Is there any reason to stick to 3.7?

### PR DESCRIPTION
**Pull request by shleeable**
Please test and verify.

Closes #228

### CODE REVIEW

**Comment by ashleyhull-versent** [comment]
Passed :)

**Comment by ashleyhull-versent** [comment]
Travis is drunk? it passed 2 hours ago.

**Comment by yosifkit** [comment]
Yeah, I was just looking at the "some checks haven't completed", yet the test is done and green when I go to Travis. :confused:

**Comment by ashleyhull-versent** [comment]
yay

File: generate-stackbrew-library.sh [hunk 1]
```diff
@@ -9,10 +9,8 @@ declare -A aliases=(
 defaultDebianSuite='stretch'
 defaultAlpineVersion='3.8'
 declare -A alpineVersion=(
-	# "openssl_missing.h:196:22: error: static declaration of 'EVP_PKEY_get0_RSA' follows non-static declaration" (and friends)
-	# https://github.com/docker-library/ruby/issues/228 / https://bugs.ruby-lang.org/issues/14754
-	[2.4]='3.7' # TODO remove this (and add Alpine 3.8) once 2.4.5+ is released
-	[2.5]='3.7' # TODO remove this (and add Alpine 3.8) once 2.5.2+ is released
+	[2.4]='3.8'
+	[2.5]='3.8'
```
Comment (tianon) [inline generate-stackbrew-library.sh]:
```suggestion
	#[2.5]='3.8'
```
(just to remind us of the syntax of this array -- and remove the `[2.4]=xxx` line entirely)

**Comment by ashleyhull-versent** [comment]
@tianon done.

**Pull request merged**